frontend/src/pages/staff-dashboard/Appoinments/AppointmentsManagement.jsx

```
@@ -1,222 +1,372 @@
+
+  // Map status from API to our internal status values
+  const mapStatusToValue = (status) => {
+    switch (status?.toLowerCase()) {
+      case 'hoàn thành':
+      case 'đã có kết quả':
+        return 'completed'
+      case 'đã xét nghiệm':
+        return 'tested'
+      case 'chờ xét nghiệm':
+      case 'chưa xét nghiệm':
+      case 'đang chờ':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)